docs/releasing/publishing-from-a-support-branch.md

```diff
@@ -0,0 +1,157 @@
+    - create release notes for the new release of GOV.UK Prototype Kit (example card: [#958](https://github.com/alphagov/govuk-prototype-kit/issues/958))
+    - create a card for the new release of GOV.UK Frontend (example card: [#1987](https://github.com/alphagov/govuk-frontend/issues/1987))
+    - update the GOV.UK Design System to use the new release of GOV.UK Frontend (example card: [#1347](https://github.com/alphagov/govuk-design-system/issues/1347))
+    - create a card for the new release of GOV.UK Prototype Kit (example card: [#917](https://github.com/alphagov/govuk-prototype-kit/issues/917))
+    - update the GOV.UK Prototype Kit to use the new release (example card: [#923](https://github.com/alphagov/govuk-prototype-kit/issues/923))
+
+3. Person leading the release to add the issues to the [Design System sprint board](https://github.com/orgs/alphagov/projects/4).
+
+4. Content designer to:
+    - write announcements to post on Slack after we release:
+      - GOV.UK Frontend (for example, [draft comms for the cookie banner component](https://docs.google.com/document/d/1jVyMB7i94NOeflWaf3kE4Q4APMXGfluK3rOh74IHO08/edit))
+      - GOV.UK Prototype kit
+    - check who the release’s contributors are and if we have consent to include their name
+
+5. Technical writer to finalise draft of release notes and get 2i on them.
+
+6. Content designer or designer to update community backlog with rationale for any decisions we made.
+
+7. Person leading the release to coordinate the sign-off when they are ready to do the release. Once the team agrees, this confirms a code and content freeze. Use the [#design-system-team-channel](https://gds.slack.com/app_redirect?channel=design-system-team-channel) to confirm sign-off from:
+    - content designer, technical writer and designers for guidance, examples and community backlog decision rationale
+    - technical writer and developers for Nunjucks macros
+    - developers for changes to GOV.UK Frontend
+    - technical writer for release notes
+    - content designer/Community manager/Technical writer for announcements and engagement activities
+
+Note: Before you go on leave, tell the delivery manager who will be looking after your work. This will help us to complete sign-off.
+
+### Publish a new version of GOV.UK Frontend from the support branch
+
+#### Change the code
+
+1. To check out the support branch for the current major release, run `git checkout support/<CURRENT MAJOR VERSION NUMBER>.x`. If the branch does not exist, follow these steps to create it:
+
+    - make sure you have all tags locally by running `git fetch --all --tags --prune`
+    - run `git checkout tags/v<LAST RELEASED VERSION NUMBER> -b support/<CURRENT MAJOR VERSION NUMBER>.x` - for example, `git checkout tags/v3.9.1 -b support/3.x`
+
+2. Run `nvm use` to make sure you’re using the right version of Node.js and npm.
```

Comment:
NPM is used in capitals earlier in the doc - we should probably choose one or the other for consistency.

Reply:
Thanks, will change the earlier mention to 'npm`, as they have it in lowercase on their site.